tests/test-dirs/locate/context-detection/cd-mod_constr.t/run.t

```diff
@@ -3,9 +3,9 @@
   {
     "class": "return",
     "value": {
-      "file": "lib/ocaml/string.ml",
+      "file": "lib/ocaml/stdlib.ml",
```

Comment:
This looks like a less useful result than before, escpecially since the user won't be able to jump to string.ml from stdlib.ml

Reply:
Interesting. Maybe we could also take into account doc comment `(** inline *)`.

Reply:
Yes, that's a shame. Another idea would be for merlin to transparently go through transparent aliases like `module Foo = Foo`. Tbh, I'm not too excited by this since people do want to jump to the alias.  Note that this isn't an issue for lsp. We have an implementation request that we could use to climb through the aliases.